lib/screenplay_web/controllers/pa_messages_api_controller.ex

```diff
@@ -20,4 +22,15 @@ defmodule ScreenplayWeb.PaMessagesApiController do
         send_resp(conn, 500, "Could not fetch audio preview")
     end
   end
+
+  def create(conn, params) do
+    case PaMessages.create_message(params) do
+      {:ok, _} ->
+        json(conn, %{success: true})
+
+      {:error, changeset} ->
+        Logger.error("[pa_message create] #{Util.format_changeset_errors(changeset)}")
```

Comment:
**note:** i'm pretty sure sentry will capture things logged at the `:error` level. we probably don't want any validation error at the API level to create a sentry issue

Reply:
Ahh, didn't think about that. Will change to `:warning`.

Reply:
Scratch that. With the new fallback controller, we shouldn't need to log anything.